index.html

```diff
@@ -105,12 +96,13 @@ <h3>DDD - CQRS - CQRS/ES</h3>
                 <small>with the permission of the author</small>
             </section>            
             <section>
-                <h3>        Domain & Ubiquitous Language</h3>
-                -
-                <h3>Two brothers always with us</h3>
-            </section>            
-            <section>
-                <h3>pic-matrioska-todo - Overall image from business to Ide and Vice Versa</h3>
+                <h3>Overall image from business to Ide and Vice Versa</h3>
```

Comment:
This doesn't matter because the H3 is all caps, but still:
```suggestion
                <h3>Overall image from business to IDE and vice versa</h3>
```